### PR TITLE
Improve the section with binding igxForOf to remote service

### PR DESCRIPTION
**Pull request by dkamburov**

### CODE REVIEW

**Comment by kdinev** [comment]
@ig-ealbert Could you check this update against your translation?

File: en/components/for_of.md [hunk 1]
```diff
@@ -107,15 +105,83 @@ The directive can be used to virtualize the data in vertical, horizontal or both
...
+    [igxForRemote]='true'
+    let-rowIndex="index" #virtDirRemote>
+    <div style='height:50px;'>{{item.ProductID}} : {{item.ProductName}}</div>
+    </ng-template>
 </div>
-<br/>
-<div class="divider--half"></div>
+```
 
+Also there is a requirement to set the `totatItemsCount` property in the instance of `igxForOf`.
```
Comment (ig-ealbert) [inline en/components/for_of.md]: this is misspelled?

Comment (ig-ealbert) [reply en/components/for_of.md]: It's "totalItemCount" right?

File: en/components/for_of.md [hunk 2]
```diff
@@ -107,15 +105,83 @@ The directive can be used to virtualize the data in vertical, horizontal or both
     </ng-template>
 </table>
 ```
+### igxFor bound to remote service
 
-**Note**: The `igxForOf` directive can bind to remote service too. Review the following demo for more information.
-#### igxFor bound to remote service Demo
-<div class="sample-container loading" style="height:780px">
-    <iframe src='{environment:demosBaseUrl}/igx-for-sample-2' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+The `igxForOf` directive can be bound to remote service. You need to use `Observeble` property - `remoteData`(in the following case). Also the `chunkLoading` event should be utilized to trigger the requests to the data.
```
Comment (ig-ealbert) [inline en/components/for_of.md]: `Observeable`?

File: en/components/for_of.md [hunk 3]
```diff
@@ -107,15 +105,83 @@ The directive can be used to virtualize the data in vertical, horizontal or both
...
+```
+And after the request to load the first chunk, the `totalItemCount` can be set:
+```typescript
+public ngAfterViewInit() {
+    this.remoteService.getData(this.virtDirRemote.state, (data) => {
+        this.virtDirRemote.totalItemCount = data["@odata.count"];
+    });
+}
+```
+When requesting data you can take advantage of `IgxForOfState` interface, which provides the `startIndex` and `chunkSize`. But note, that initialy the chunkSize will be 0, so you have to specify the size of the first loaded chunk(the best value is `igxForContainerSize` divided on `igxForItemSize`).
```
Comment (ig-ealbert) [inline en/components/for_of.md]: initially
"divided on"?  "divided by"?

File: en/components/for_of.md (hunk 2 above)
Comment (dkamburov) [reply en/components/for_of.md]: Should be Observable, it's a typo

File: en/components/for_of.md (hunk 3 above)
Comment (dkamburov) [reply en/components/for_of.md]: "the best value is `igxForContainerSize` initially divided by `igxForItemSize` by sounds better I think

File: en/components/for_of.md (hunk 1 above)
Comment (dkamburov) [reply en/components/for_of.md]: Yes it is misspelled.

**Pull request merged**